hogben/optimise.py

```diff
@@ -12,6 +12,64 @@
     VariableContrast,
     VariableUnderlayer,
 )
+from hogben.utils import Fisher, sig_fig_round
+from hogben.visualise import scan_parameters
+
+
+def optimise_parameters(sample: BaseSample,
```

Comment:
I think you have different formatting/indentation for these arguments to all other functions in the file